CRM/Utils/Geocode/Google.php

```diff
@@ -121,10 +121,11 @@ public static function format(&$values, $stateName = FALSE) {
 
     $query = 'https://' . self::$_server . self::$_uri . $add;
 
-    require_once 'HTTP/Request.php';
-    $request = new HTTP_Request($query);
-    $request->sendRequest();
-    $string = $request->getResponseBody();
+    $response = CRM_Utils_Http::get($query);
```

Comment:
Actually - if you were going to play with any part of this in an extension this would be a good place - since that address validation needs to be extensionised at some point